core/src/components/spinner/spinner.ionic.scss

```diff
@@ -11,6 +11,10 @@
   height: globals.$ion-scale-700;
 }
 
+:host(.ion-color.ion-color-medium) {
+  color: globals.current-color(contrast, null, true);
```

Comment:
This doesn't seem right to me. I don't think we should be overriding colors by targeting them like this. Additionally, [this is already the default color for the host](https://github.com/ionic-team/ionic-framework/pull/30418/files#diff-04156e01912c869b309107a345cfd4191bfee7223b4a66b7ad0260b13f5aa5c3L8) (this color - medium subtle - evaluates to be the same value), so this is unnecessary.

Reply:
I can iterate on this to simplify the selector, however, I think I have to keep it. Because the pretended color for medium palette is contrast-subtle and not -ion-color-medium 

![spinnercolors](https://github.com/user-attachments/assets/7a558678-cf08-4d44-bdb6-bcb6a0da3118)